trieste/acquisition/function.py

```diff
@@ -908,10 +935,19 @@ def prepare_acquisition_function(
         mean, _ = model.predict(dataset.query_points)
 
         _pf = Pareto(mean)
-        _reference_pt = get_reference_point(_pf.front)
+        if isinstance(self._ref_point_specification, FunctionType):
```

Comment:
FunctionType is only for user defined functions so won't work for e.g. class instances with a call method (or built in functions).

```suggestion
        if callable(self._ref_point_specification):
```